spec/http-gateway-protocol-spec.md

```diff
@@ -448,6 +448,19 @@ You can also [download the file](./_attachments/http-gateway.did).
 
 Not all of this interface is required. The following sections detail what can be optionally omitted depending on the requirements of the canister in question.
 
+Note. Composite query methods can be used instead of query methods to allow for calling composite query methods of other canisters on the same subnet
+when processing an HTTP request, e.g., the canister can export
+
+```
+http_request: (request: HttpRequest) -> (HttpResponse) composite_query;
```

Comment:
Does this require any changes in the HTTP Gateway implementations to leverage this?

Reply:
As long as the HTTP gateway implementations make HTTP requests to `/api/v2/.../query` to call this method, no changes are necessary. But note that HTTP requests to `/api/v2/.../call` for composite queries will be rejected (I'd expect the HTTP gateway to use the other method http_request_update if an update call is to be made).

Reply:
Thanks for the clarification. Yes, the HTTP Gateway won't make an update call to `http_request`, only to `http_request_update`.